evil-commands.el

```diff
@@ -2924,6 +2924,26 @@ for the last window in each frame."
                   (error nil)))
             wins))))
 
+(evil-define-command evil-wipeout-buffer (buffer &optional bang)
+  "Deletes a buffer. Also clears related jump marks.
+All windows currently showing this buffer will be closed except
+for the last window in each frame."
+  (interactive "<b><!>")
+  (let* ((ring (make-ring evil-jumps-max-length))
+         (struct (evil--jumps-get-current))
+         (idx (evil-jumps-struct-idx struct))
```

Comment:
What is this variable even used for?

Reply:
it's the updated index of the new jump ring